packages/turf-constrained-tin/test.js

```diff
@@ -0,0 +1,75 @@
+const fs = require('fs');
+const path = require('path');
+const test = require('tape');
+const constrainedTin = require('./');
+
+const points = require(path.join(__dirname,  'test', 'Points.json'));
+const points_grid = require(path.join(__dirname,  'test', 'Points_grid.json'));
+const mercator = require(path.join(__dirname,  'test', 'mercator.json'));
+
+test('tin - z property', t => {
+    const expected = require(path.join(__dirname,  'test', 'Tin.json'));
```

Comment:
Sorry, I only noticed this now. Each turf package uses a similar pattern here, which is dividing test input and expected results into `in` and `out` folders (see [here](https://github.com/Turfjs/turf/tree/master/packages/turf-circle/test) for example). This makes the test code [simpler](https://github.com/Turfjs/turf/blob/master/packages/turf-circle/test.js#L25) and easier to add new test cases if necessary.

Reply:
OK, I will divide 'in' and 'out'.

Reply:
Done